pkgs/top-level/all-packages.nix

```diff
@@ -37203,6 +37203,8 @@ with pkgs;
   dump = callPackage ../tools/backup/dump { };
 
   dxvk = callPackage ../misc/dxvk { };
+  dxvk_1 = callPackage ../misc/dxvk/dxvk.nix { dxvkVersion = "1.10"; };
+  dxvk_2 = callPackage ../misc/dxvk/dxvk.nix { dxvkVersion = "2.1"; };
```

Comment:
What is the difference between dxvk and dxvk_2?

Reply:
Ah, I see. The script is added only in dxvk, and there version is chosen automatically. Maybe describe it somewhere, like in dxvk description.

Reply:
`dxvk_1` and `dxvk_2` are the native libraries. `dxvk` is the derivation with the setup script. It was originally the only one, but with the merging of native support in 2.0, I wanted to provide Linux builds without breaking the previous derivation. This also allows for users with older hardware to continue using `dxvk_1` if that works better for them.

I pushed a commit to update the description on `dxvk` to indicate that it’s the setup script.